pickee_api/gateway.py

```diff
@@ -209,6 +209,18 @@ def generate_recommendation(request):
         discoverResponse = requests.get(url, params=query_params, auth=BearerAuth(TMDB_ACCESS_TOKEN))
         discoverData = discoverResponse.json()
 
+        # Removes actors from the query if there are no results
+        if len(discoverData.get('results')) == 0:
+            del query_params['with_cast']
+            discoverResponse = requests.get(url, params=query_params, auth=BearerAuth(TMDB_ACCESS_TOKEN))
+            discoverData = discoverResponse.json()
+        
+        # Removes keywords from the query if there are still no results
+        if len(discoverData.get('results')) == 0:
```

Comment:
@rjs68 @Arden488 
Discussion: what is better? To get a random movie or a modal alert telling me that I was too picky?

Reply:
The first one! We still take the genres into account and I think this is a very rare situation anyway. It's not one I've actually come across, it's just in there as a just in case scenario

Reply:
Okay!